File: iOS/Matcha/Matcha/Sources/Scenario.swift
//
//  Testcase.swift
//  Vicky
//
//  Created by Andre Guggenberger on 10/05/2017.
//  Copyright © 2017 CASEapps. All rights reserved.
//

import Foundation

class Scenario {
    var name : String
    var preconditions:[Action] = []
    var steps:[Action] = []
    
    init(name: String) {
        self.name = name
    }
}

File: Android/Matcha/app/src/main/java/at/caseapps/matcha/Scenario.java
package at.caseapps.matcha;

import java.util.ArrayList;
import java.util.List;

public class Scenario {
    public String name;
    public List<Action> preconditions = new ArrayList<>();
    public List<Action> steps = new ArrayList<>();

    public Scenario(String name) {
        this.name = name;
    }
}
